config/routes.rb

```diff
@@ -0,0 +1,26 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root to: "users#new"
+
+  resources :users do 
```

Comment:
i like your decision to put timeline, friend, and photo as separate singular resources and just use show for each route. cleaner than my implementation